main.go

```diff
@@ -249,23 +255,22 @@ func installLatestImplicitVersion(requestedVersion string, custBinPath, mirrorUR
 		fmt.Printf("error parsing constraint: %s\n", err)
 	}
 	//if lib.ValidMinorVersionFormat(requestedVersion) {
-	tfversion, err := lib.GetTFLatestImplicit(*mirrorURL, preRelease, requestedVersion)
-	if err == nil && tfversion != "" {
-		lib.Install(tfversion, *custBinPath, *mirrorURL)
+	tfRelease, err := lib.GetTFLatestImplicit(*mirrorURL, preRelease, requestedVersion)
+	if err == nil && tfRelease.Version != "" {
+		lib.Install(tfRelease, *custBinPath)
 	}
 	fmt.Printf("Error parsing constraint: %s\n", err)
 	lib.PrintInvalidMinorTFVersion()
 }
 
 // show latest - argument (version) must be provided
-func showLatestImplicitVersion(requestedVersion string, custBinPath, mirrorURL *string, preRelease bool) {
+func showLatestImplicitVersion(requestedVersion string, mirrorURL *string, preRelease bool) {
 	if lib.ValidMinorVersionFormat(requestedVersion) {
 		tfversion, _ := lib.GetTFLatestImplicit(*mirrorURL, preRelease, requestedVersion)
```

Comment:
Probably want to check this error?
The following line is liable to get a nil pointer error if it fails.

Reply:
Thanks, added